bin/cli.js

```diff
@@ -40,6 +41,10 @@ const browserLaunchers = {
   'Safari':    'karma-safari-launcher'
 };
 
+const logLevels = [ 'error', 'warn', 'info', 'debug' ];
+
+const isLogAtLeast = (verbosity, level) => verbosity >= logLevels[level];
```

Comment:
How does this function work?
According to its usage, `level` argument is a string naming logging level, and `logLevels` is an array that doesn't seem to map the name of the logging level to its verbosity value.